ports/stm32/usbd_cdc_interface.h

```diff
@@ -43,8 +43,8 @@ typedef struct _usbd_cdc_itf_t {
     uint16_t rx_buf_get; // circular buffer index
 
     uint8_t tx_buf[USBD_CDC_TX_DATA_SIZE]; // data for USB IN endpoind is stored in this buffer
-    uint16_t tx_buf_ptr_in; // increment this pointer modulo APP_TX_DATA_SIZE when new data is available
-    volatile uint16_t tx_buf_ptr_out; // increment this pointer modulo APP_TX_DATA_SIZE when data is drained
+    uint16_t tx_buf_ptr_in; // increment this pointer modulo USBD_CDC_RX_DATA_SIZE when new data is available
+    volatile uint16_t tx_buf_ptr_out; // increment this pointer modulo USBD_CDC_RX_DATA_SIZE when data is drained
```

Comment:
Both should actually be USBD_CDC_TX_DATA_SIZE.

Reply:
Sorry for the mistake.:blush: